File: java/src/main/java/com/iteso/nintendo/Hadouken.java
package com.iteso.nintendo;

/**
 * Created by rvillalobos on 2/24/18.
 */
public class Hadouken implements Attack {
    private int attackDamage;

    public String attack() {
        return "Hadouken";
    }

    public void setAttackDamage(int attackDamage) {
        this.attackDamage = attackDamage;
    }

    public int getAttackDamage() {
        return attackDamage;
    }
}

File: csharp/Clases/Attacks/Hadouken.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace unittestpractice.Clases.Attacks
{
    public class Hadouken : IAttack
    {
        /**
         * attack damage.
         */
        private int attackDamage;

        /**
         * @return attack name.
         */
        public String attack()
        {
            return "Hadouken";
        }

        /**
         * @return attack damage.
         */
        public int getAttackDamage()
        {
            return attackDamage;
        }

        /**
         * @param damage attack damage.
         */
        public void setAttackDamage(int damage)
        {
            attackDamage = damage;
        }
    }
}
